src/react-components/preferences-screen.js

```diff
@@ -390,6 +392,10 @@ const preferenceLabels = defineMessages({
     id: "preferences-screen.preference.disable-auto-gain-control",
     defaultMessage: "Disable microphone automatic gain control"
   },
+  automaticOutputGainControl: {
+    id: "preferences-screen.preference.audio-automatic-output-gain-control",
+    defaultMessage: "[Experiment] Automatic Output Gain Control"
```

Comment:
Suggestion for wording and capitalization to match the existing preferences:
```suggestion
    defaultMessage: "Automatic incoming voice volume control (experimental)"
```

Reply:
Looking good to me. Thanks.